src/test/java/org/folio/edge/core/EdgeVerticleTlsIntegrationTest.java

```diff
@@ -0,0 +1,92 @@
+package org.folio.edge.core;
+
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpServer;
+import io.vertx.core.http.HttpServerOptions;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.net.KeyStoreOptions;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import io.vertx.ext.web.client.WebClient;
+import io.vertx.ext.web.client.WebClientOptions;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.bouncycastle.jcajce.provider.BouncyCastleFipsProvider;
+import org.folio.edge.core.utils.SslConfigurationUtil;
+import org.folio.edge.core.utils.test.TestUtils;
+import org.junit.*;
+import org.junit.runner.RunWith;
```

Comment:
first line don't need I think

Reply:
Do you mean remove I need to remove `import org.junit.*;`?

Reply:
need use specific imports instead of implicit import style